### PR TITLE
Update Home Page Content

### PR DESCRIPTION
**Pull request by EduardoLR10**

### CODE REVIEW

File: content/english/_index.md [hunk 1]
```diff
@@ -3,25 +3,29 @@
...
+  - title: "Quality as Paramount"
+    image: "/images/service-1.png"
+    content: "In our projects, we merge practical expertise in functional programming with a commitment to delivering solutions on time and within budget --- you can trust us that your product will fly in no time. As a boutique, we focus on efficient delivery at low cost, without compromising on quality and transparency. Our team of dedicated professionals leverages both practical and abstract strategies to create reliable software solutions that meet our clients' requirements.  We value partnerships with the open-source community, fostering an environment where diverse perspectives converge to shape tailored solutions that best fit the product."
```
Comment (MMagueta) [inline content/english/_index.md]: The `as a boutique we <something>` feels off. It begs for a justification of why that is the case as a boutique. I would just remove it or change `boutique` for something else

**Comment by MMagueta** [comment]
Overall it's good, I just feel like it is somewhat hollow. It does sound like an annoying seller's speech. I think it lacks empathy, which is hard to build on offering contractual services. How would DrN fit in a story? Specifically of this known form:

- Hero :: Hiring manager
- Problem
  + External :: my people are leaving
  + Internal :: I am anxious to fill up these postions
  + Philosophical :: we are google, people *should* bang at our door
  + Villain :: Smug geek
- Mentor ::
  - Authority :: CTO, experience
  - Sympathy ::  I like your products, and grew up with you
- Process ::
  + Understand my growth dev->manager
- Promises ::
  +

- Success :: Product organically growing
- Failure :: ad-hoc actions leading to being stuck

File: content/english/_index.md [hunk 2]
```diff
@@ -3,25 +3,29 @@
...
       label: "Get to know Us"
       link: "team"
 
-  - title: "Efficiency at the best Price"
-    image: "/images/service-1.png"
-    content: "Welcome to Don't rely on Nulls, where we merge practical expertise in functional programming with a commitment to delivering solutions on time and within budget. As a boutique, we focus on efficient delivery at low cost, without compromising on quality and transparency. Our team of dedicated professionals values both practical and abstract strategies to create reliable software solutions that meet our clients' requirements."
+  - title: "Humility and Learning"
+    image: "/images/service-2.png"
+    content: "Integrity is paramount to our approach. We adhere to the highest ethical standards and craftsmanship to ensure our solutions align seamlessly with our clients' objectives. In a rapidly evolving tech landscape, we remain humble learners, valuing grounded ideas and the pursuit of improvement. Our team embraces ongoing education and exploration to shape the future of software development. Collaboration is at the heart of our ethos. Our hobby projects demonstrate how dedicated we are into exploration and facing uncharged territories."
```
Comment (z-silver) [inline content/english/_index.md]:
```suggestion
    content: "Integrity is key to our approach. We adhere to the highest ethical standards and craftsmanship to ensure our solutions align seamlessly with our clients' objectives. In a rapidly evolving tech landscape, we remain humble learners, valuing grounded ideas and the pursuit of improvement. Our team embraces ongoing education and exploration to shape the future of software development. Collaboration is at the heart of our ethos. Our hobby projects demonstrate how dedicated we are to braving uncharted territories."
```

Brevity is wit. Also, it should be "uncharted".

File: content/english/_index.md [hunk 3]
```diff
@@ -3,25 +3,29 @@
 banner:
   title: "Don't rely on Nulls, rely on us"
   content: 
-    "Allow us to make your dream a reality with our unmatchable attention to detail and elegance praising."
+    "Fear of unreliable software? Rest and allow us to make your dream a reality with our unmatched attention to detail and diligence towards reputable software"
```
Comment (z-silver) [inline content/english/_index.md]: The "rest" bit feels off. It really doesn't sound like something a native speaker would say. Maybe "rest easy" or "relax" instead?

Comment (z-silver) [inline content/english/_index.md]:
```suggestion
    "Fear of unreliable software? Rest and allow us to make your dream a reality with our unmatched attention to detail and diligence towards reputable software."
```

File: content/english/_index.md [hunk 4]
```diff
@@ -3,25 +3,29 @@
...
+    button:
+      enable: true
+      label: "Check Projects"
+      link: "projects"
 
   - title: "Business Value via FP"
     image: "/images/service-3.png"
-    content: "Choose Don't rely on Nulls for dependable software solutions. Join us on a journey where practical innovation meets philosophical reflection, guided by our expertise in functional programming, towards a brighter digital horizon. The outcome will be pretty and elegant enough that you will want to hang on your wall."
+    content: "Choose Don't rely on Nulls for dependable software solutions. Join us on a journey where practical innovation meets from-first-principles reflection, guided by our expertise in functional programming, towards a brighter digital horizon. The outcome will be pretty, and maintanable enough that you will like to frame it."
```
Comment (z-silver) [inline content/english/_index.md]:
```suggestion
    content: "Choose Don't rely on Nulls for dependable software solutions. Join us on a journey where practical innovation meets reflection from first principles, guided by our expertise in functional programming, towards a brighter digital horizon. The outcome will be pretty, and maintanable enough that you'd like to frame it."
```

The "will" there feels rather forceful, more like a command than a statement of fact. You can drop the contraction and say "would", but that doesn't read as smoothly.

I don't see the point of a hyphenated adjective here either. The normal way of saying it works just fine.

File: content/english/_index.md [hunk 5]
```diff
@@ -3,25 +3,29 @@
...
+  - title: "Quality as Paramount"
+    image: "/images/service-1.png"
+    content: "In our projects, we merge practical expertise in functional programming with a commitment to delivering solutions on time and within budget --- you can trust us that your product will fly in no time. We focus on efficient delivery at low cost, without compromising on quality and transparency. Our team of dedicated professionals leverages both practical and abstract strategies to create reliable software solutions that meet our clients' requirements. We value partnerships with the open-source community, fostering an environment where diverse perspectives converge to shape tailored solutions that best fit the product."
```
Comment (z-silver) [inline content/english/_index.md]: The "fly in no time" bit sounds redundant, after we just said we were committed to delivering solutions on time. I'm not quite sure how to change it, though. A good candidate is "soar", but that sounds slightly pretentious in this context. I'm not sure that we would want that while simultaneously claiming to be humble. Also good is "shine", but that has no connotation of height.

Comment (z-silver) [reply content/english/_index.md]: Perhaps "will soon fly"?

File: content/english/_index.md (hunk 3 above)
Comment (EduardoLR10) [reply content/english/_index.md]: A subtitle ending with dot feels a little bit off.

**Review by z-silver — APPROVED** [review]

**Pull request merged**